### PR TITLE
Add MIW videos to blog posts

### PR DESCRIPTION
**Pull request by scottslowe**
This PR updates four blog posts to add relevant/related Modern Infrastructure Wednesday (MIW) videos via the `youtube` shortcode.

## Description

This blog embeds related MIW videos into four blog posts:

- Building a data warehouse on AWS with Redshift and Pulumi
- Create and Manage Confluent Kafka Cluster with Pulumi
- How to Create and Share a Pulumi Template
- Redshift ETL with Pulumi and AWS Glue

Merging this PR will close https://github.com/pulumi/devrel-team/issues/542

## Checklist:

- [x] I have reviewed the [style guide](https://github.com/pulumi/pulumi-hugo/blob/master/STYLE-GUIDE.md).
- [x] If blogging, I have reviewed the [blogging guide](https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md).
- [x] I have manually confirmed that all new links work.
- [ ] I added aliases (i.e., redirects) for all filename changes.

### CODE REVIEW

File: themes/default/content/blog/building-a-data-warehouse-on-aws-with-redshift-and-pulumi/index.md [hunk 1]
```diff
@@ -457,6 +457,10 @@ SELECT *
 
 Huzzah! You've now got yourself a working data warehouse.
 
+Here's a [Modern Infrastructure](https://www.youtube.com/playlist?list=PLyy8Vx2ZoWloyj3V5gXzPraiKStO2GGZw) video that demonstrates everything you've just read in this blog post:
```
Comment (susanev) [inline themes/default/content/blog/building-a-data-warehouse-on-aws-with-redshift-and-pulumi/index.md:460]: can we put video into the link text?

Comment (susanev) [inline themes/default/content/blog/building-a-data-warehouse-on-aws-with-redshift-and-pulumi/index.md:460]:
```suggestion
Here's a [Modern Infrastructure video](https://www.youtube.com/playlist?list=PLyy8Vx2ZoWloyj3V5gXzPraiKStO2GGZw) that demonstrates everything you've just read in this blog post:
```

**Review by susanev — APPROVED** [review]

File: themes/default/content/blog/building-a-data-warehouse-on-aws-with-redshift-and-pulumi/index.md (hunk 1 above)
Comment (scottslowe) [reply themes/default/content/blog/building-a-data-warehouse-on-aws-with-redshift-and-pulumi/index.md:460]: @susanev For sure! Would you like to replicate that change to the other blog posts in this PR as well?

Comment (susanev) [reply themes/default/content/blog/building-a-data-warehouse-on-aws-with-redshift-and-pulumi/index.md:460]: yah! wasnt sure if it was easier for you for me to repeat a suggested change. would you like me to?

Comment (scottslowe) [reply themes/default/content/blog/building-a-data-warehouse-on-aws-with-redshift-and-pulumi/index.md:460]: Nah, no need. I'll make the same change elsewhere.

**Review by cnunciato — APPROVED** [review]
Approving for now as this gets the job done, but part of me wishes there were a way to do this without just dropping the videos into the middle of the narrative. It'd be better, I think, if we could come up with a way to surface them in more of a "see also" sort of context, maybe with a spot for some additional description (and perhaps even other "related" things, like examples, etc.). What do you think? Maybe worth a follow-up issue if you agree?

**Comment by scottslowe** [comment]
@cnunciato If there's a better approach to take, I'm happy to rework this PR with that approach. What are you thinking? Another H2 that says "More Resources" or "See a Video Walkthrough"?

**Comment by scottslowe** [comment]
@cnunciato I added a heading and a bit more text to help with the "dropping it in the middle" feeling, but I do agree it would be great to have some sort of "related things" mechanism for blog posts that could link to MIW videos, examples, templates, etc. I suppose that is the job of a CMS! I'll create a follow-up item for us to explore options.

**Pull request merged**